package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "schule",
+  "version": "1.0.0",
+  "description": "",
+  "main": "main.js",
+  "scripts": {
+    "test": "gulp",
+    "start": "npm rn server",
+    "server": "cd app/server && tsc main.ts --outDir dist --lib ES2015 && cd dist/server && node main.js",
```

Comment:
In the refactor, consider using tsc itself and configure the rest in tsconfig.json. With complexity of the app only compiling from the main.ts onwards might not be a clean solution anymore.

Same for the client instruction.